classes/person.rb

```diff
@@ -1,15 +1,22 @@
 require_relative 'nameable'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Great job so far in defining the associations for your `Person` class! 👏 However, I'd like to highlight a small adjustment regarding the use of your attributes and suggest some changes to your `setters` and `getters`.

    - In your `Person` class, consider using `attr_accessor` for the `rentals` attribute instead of `attr_reader`. This change will allow other classes to not only read the `rentals` attribute from the `Person` class but also modify its value when needed. This can be particularly useful for scenarios where you may need to update or change a person's rentals. 😊

Happy coding! 😊🚀🍀

Reply:
✅ Done